### ISSUE TITLE
Not working

### ISSUE THREAD

**Issue by anshulsolanki** [bhavaniravi/dialogflow_to_rasa_core#1]
This not working, the NLU part is correct, but domain file is not correct

**Comment by bhavaniravi** [comment]
hey, @anshulsolanki  thanks for trying it out. This is just the first iteration need some testing. I am testing it with Dialogflow predefined logs any inputs on logs would help. 

Also please tryout the recent push.

**Comment by anshulsolanki** [comment]
Thanks for the update. I tested and it is working now. But with 1 problem, i.e.
you are not putting quotes in all text messages, i.e.
templates:
  affirm:
  - text: "Me alegro de ayudarte, compa\xC3\xB1ero :)"
  - text: Glad I help you, mate :)
  goodbye:
  - text: "\xC2\xA1Nos vemos! Disfrutar"
  - text: See ya! Enjoy

--------------------
here the line
- text: Glad I help you, mate :)
should be 
- text: "Glad I help you, mate :)"


I manually changed that in domain file , doing that i can train my NLU and dialogue models.

Fix this and it should be fine.

After your fix i will test again.

Thanks, good work!!!
